facebook/OpenBIC
# common: ipmi: Add get bic info command

Summary:
- Modify platform name to mach sign tool
- Add get bic info command NetFn:0x38 Cmd:0x0A, It's for **BIC sign-image update** use. 
  - Request:\
    [Byte 1:3] MFG ID\
    [Byte 4:4] option\
	       0x00 - get bic platform name\
	       0x01 - get bic board id\
	       0x02 - get bic baord name\
	       0x03 - get bic board stage
  - Response:\
    [Byte 1:1] Completion Code
    [Byte 2:4] MFG ID
    [Byte 5:?] fw info
               EX: **Waimea Canyon mainboard evt board**
               - platform name (Waimea Canyon)\
                 57 61 69 6D 65 61 20 43 61 6E 79 6F 6E\
               - board id (01: mainboard)\
                   01\
               - board name (mainboard)\
                 6D 61 69 6E 62 6F 61 72 64\
               - board stage (00:poc 01:evt 02:dvt 03:pvt 04:mp)\
                 01
- Get bic info command supports for both BMC and HOST.

Test Plan:
- Build Code: PASS
- Check command via BMC/HOST: PASS

Log:
- BMC console:
  ```
  root@bmc-oob:~# bic-util mb 0xe0 0x0a 0x15 0xa0 0x00 0x01
  15 A0 00 57 61 69 6D 65 61 20 43 61 6E 79 6F 6E 
  root@bmc-oob:~# bic-util mb 0xe0 0x0a 0x15 0xa0 0x00 0x02
  15 A0 00 01 
  root@bmc-oob:~# bic-util mb 0xe0 0x0a 0x15 0xa0 0x00 0x03
  15 A0 00 6D 61 69 6E 62 6F 61 72 64 
  root@bmc-oob:~# bic-util mb 0xe0 0x0a 0x15 0xa0 0x00 0x04
  15 A0 00 01
  ```

- HOST console:
  ```
  [root@localhost ~]# ipmitool raw 0x38 0x0a 0x15 0xa0 0x00 0x01
  15 a0 00 57 61 69 6d 65 61 20 43 61 6e 79 6f 6e
  [root@localhost ~]# ipmitool raw 0x38 0x0a 0x15 0xa0 0x00 0x02
  15 a0 00 01
  [root@localhost ~]# ipmitool raw 0x38 0x0a 0x15 0xa0 0x00 0x03
  15 a0 00 6d 61 69 6e 62 6f 61 72 64
  [root@localhost ~]# ipmitool raw 0x38 0x0a 0x15 0xa0 0x00 0x04
  15 a0 00 01
  ```
